### PR TITLE
Added quick fix for "tab before keyword" warning.

### PR DESCRIPTION
**Pull request by radhikaaggarwal19**
[Added](https://github.com/microsoft/vscode-flake8/issues/41) quick fix for "tab before keyword" warning

### CODE REVIEW

File: bundled/tool/server.py [hunk 1]
```diff
@@ -236,7 +236,7 @@ def code_action(params: lsp.CodeActionParams) -> List[lsp.CodeAction]:
     return code_actions
 
 
-@QUICK_FIXES.quick_fix(codes=["E275"])
+@QUICK_FIXES.quick_fix(codes=["274", "E275"])
```
Comment (karthiknadig) [inline bundled/tool/server.py:239]: This is missing the E
```suggestion
@QUICK_FIXES.quick_fix(codes=["E274", "E275"])
```

**Review by radhikaaggarwal19 — COMMENTED** [review]
Edited for missing 'e'

**Comment by karthiknadig** [comment]
```
nox > black --check ./src/test/python_tests
would reformat src/test/python_tests/test_code_actions.py
```

@radhikaaggarwal19 looks like there is some formatting issue with `test_code_actions.py`. Can you install `black` in your environment (`python -m pip install black`), and then try running this `python -m black src/test/python_tests/test_code_actions.py`. After that commit those changes and push.

**Comment by radhikaaggarwal19** [comment]
I made the update and committed the changes. Hope it works now!

**Pull request merged**